packages/pixeloven/cli-addon-webpack/src/commands/webpack.ts

```diff
@@ -33,19 +52,82 @@ export default {
         switch (task) {
             case "build":
             case "start": {
+                if (!client && !library && !server) {
+                    pixelOven.invalidArgument(
+                        `Please provide an entry for Webpack to target. Available options are "--client", "--library", or "--server".`,
+                    );
+                    pixelOven.exit("Webpack", 1);
+                    return;
+                }
+
                 /**
                  * @todo Need to list all options in some sorta help style menu
                  */
                 Object.keys(parameters.options).forEach(option => {
                     if (!WebpackExecutionOptionTypes.hasOwnProperty(option)) {
                         pixelOven.invalidArgument(
-                            `Available options for "${task}" are "--path", "--source-map", or "--stats"`,
+                            `Available options for "${task}" are "--client", "--development", "--entry", "--host", "--ignored", "--library", "--path", "--port", "--protocol", "--server", "--source-map", or "--stats"`,
                             `--${option}`,
                         );
                         pixelOven.exit("Webpack", 1);
                     }
                 });
 
+                const breakOption = (option: string | boolean) => {
+                    if (typeof option === "boolean") {
+                        return {};
+                    } else {
+                        const options = option.includes(":")
+                            ? option.split(":")
+                            : [option];
+                        if (options.length > 1) {
+                            return {
+                                entry: options[1],
+                                target: Target[options[0]],
+                            };
+                        } else if (options[0].includes(".")) {
+                            return {
+                                entry: options[0],
+                            };
+                        } else {
+                            return {
+                                target: Target[options[0]],
```

Comment:
Should probably check that the Target key is supported and throw an error here if it's not.